cmake/tests.cmake

```diff
@@ -16,7 +16,8 @@ if(${BUILD_TEST})
       ${TENSEAL_TESTS_BASEDIR}/tensors/ckksvector_test.cpp
       ${TENSEAL_TESTS_BASEDIR}/tensors/ckkstensor_test.cpp
       ${TENSEAL_TESTS_BASEDIR}/tensors/plaintensor_test.cpp
-      ${TENSEAL_TESTS_BASEDIR}/tensors/bfvvector_test.cpp)
+      ${TENSEAL_TESTS_BASEDIR}/tensors/bfvvector_test.cpp
```

Comment:
Why the duplicate cpp file?

Reply:
No sorry probably I just copy pasted and changed again after having saved, those lines should be appearing:
```cmake
${TENSEAL_TESTS_BASEDIR}/tensors/bfvvector_test.cpp
${TENSEAL_TESTS_BASEDIR}/tensors/bfvtensor_test.cpp)
```